wrappers/wrapper.rb

```diff
@@ -215,6 +215,16 @@ def assert_square_matrix(vrp)
       }
     end
 
+    def assert_correctness_provided_matrix_indices(vrp)
+      dimensions = vrp.vehicles.collect(&:dimensions).flatten.uniq
+      max_matrix_index = vrp.points.collect(&:matrix_index).max || -1
+      vrp.matrices.all?{ |matrix|
+        dimensions.all?{ |dimension|
+          matrix[dimension].nil? || matrix[dimension].size > max_matrix_index && matrix[dimension].all?{ |line| line.size > max_matrix_index }
+        }
+      }
+    end
+
```

Comment:
@fonsecadeline this assert removed recently because it is handled in method `vrp.create` now (since it was an assert for all solvers, it is actually not an assert).

I feel like there was a rebase or conflict management mistake :thinking:

Reply:
It is indeed, thank you for noticing